cime_config/testdefs/testlist_clm.xml

```diff
@@ -2033,13 +2033,13 @@
       <option name="comment">60 day exact restart test providing coverage for the FATES logging mode on an f45 grid.</option>
     </options>
   </test>
-  <test name="ERS_Ld30" grid="f45_f45_mg37" compset="I2000Clm50FatesRs" testmods="clm/FatesColdDefSizeAgeMort">
+  <test name="ERS_Ld30" grid="f45_f45_mg37" compset="I2000Clm50FatesRs" testmods="clm/mimicsFatesColdDefSizeAgeMort">
```

Comment:
The ColdDefSizeAgeMort is a very specific way to run FATES with a tree mortality function added by @JessicaNeedham. 

I like the grid and that this is an ERS_Ld30.  But perhaps we should create a new test, and base the configuration off of FatesColdDef and not FatesColdDefSizeAgeMort.  Otherwise, we will have trouble disentangling any failure attribution to the size-age-mort stuff vis-a-vis the mimics coupling.

Also, we should give this an aux_clm-izumi, and a category="fates" with cheyenne.

@slevisconsulting , others, sound good?

Reply:
That makes sense to me @rgknox. This is an example of why we wanted you to look at this.

Reply:
@rgknox @ekluzek 
I reverted the existing test to the way it was and added the recommended test as
`name="izumi" compiler="intel" category="aux_clm"`

Were you recommending that I repeat this test on cheyenne with category="fates"? I'm double checking to avoid unnecessary proliferation of tests :-)

Reply:
@slevisconsulting I think this is one thing that is worth doing. Change the name of the testmod and also remove the setting for size-age-mort stuff. So it just turns on mimics. And then redo the tests that have this test mod.

With that this should be ready to go. So let me know when you have that ready and I can make it into a tag.

Reply:
@ekluzek I'm pretty sure you're talking about a change that I made in https://github.com/ESCOMP/CTSM/pull/1643/commits/d95a71623115455b223e3aa5387a646045dc757c
where I backed my changes out of any existing tests.

If you look at the cumulative changes from all commits in this PR and search for testlist_clm.xml, you'll see only the new test that we talked about today.

Reply:
Ahhh, OK. For some reason looking at it here, I was seeing the earlier commit and not the latest. The latest looks fine. Thanks for doing that renaming.